client/libra-dev/README.md

```diff
@@ -0,0 +1,87 @@
+* Creating a `K`-of-`N` [multisig](https://github.com/libra/libra/blob/master/crypto/crypto/src/multi_ed25519.rs) authentication key is similar: generate `N` ed25519 public keys, then compute `auth_key = (<concatenated_keys> | K | 1)`. Derive an address and an auth key prefix as described above.
+* Libra supports key rotation via changing the `auth_key` stored under an account. The address associated with an account never changes.
+
+## Exercising New Functionality in Testnet
+
+### Creating accounts and minting with the faucet
+
+As in the original Libra, testnet has a faucet service for creating accounts and giving money to existing accounts. There are three important changes from the original faucet:
+
+* The faucet now [needs](https://github.com/libra/libra/pull/3972) one of the currency codes above
+* The faucet can accept either a 16 byte address (for an existing account), a 32 byte account authentication key (for an account that does not yet exist and should be created by the faucet). Previously, account addresses and authentication keys were both 32 byte values, but account addresses have [shrunk](https://github.com/libra/libra/issues/2764).
+* An account created by the faucet has the `ParentVASP` role. Note that in mainnet, only the Association will have the privilege to create `ParentVASP`s, and will do so only for entities that meet appropriate standards.
+
+### Transactions
+
+Transactions are mostly unchanged from the original design. Two notable differences are:
+
+* Transactions now require a [`gas_currency_code`](https://github.com/libra/libra/blob/master/types/src/transaction/mod.rs#L72) specifying which currency should be used to pay for gas. The account must have a balance in this currency.
+* The signature scheme for a transaction can be either be single-signature ed25519 (as the original design) or multi-ed25519, a new [multisig](https://github.com/libra/libra/issues/2431) format
+
+### Payments
+
+Payments can be sent using [this](https://github.com/libra/libra/blob/master/language/stdlib/transaction_scripts/peer_to_peer_with_metadata.move) transaction script. There are a few changes:
+
+* The script requires a generic type parameter `Token` specifying the currency to be transferred. The sending and receiving account must both have a balance in this currency. A transaction that attempts to send (e.g.) `LBR` to an account that doesn’t have an `LBR` balance will abort.
+* The script has a `metadata` parameter that accepts arbitrary binary data. For most transactions, the metadata should be the [subaddress](https://docs.google.com/document/d/1TzPQ6oRgYLvudpojSa424L6pvFOQGqecD9bPCrEyV-U/edit?ts=5ea7b737#) of the VASP customer receiving the payment. The contents of `metadata` are emitted in payment events, but are not otherwise inspected on-chain (so using empty or dummy `metadata` is just fine for testing).
```

Comment:
Not sure if the google doc link should be kept in since that will be dead for people without access to it?